conan/tools/cmake/cmakedeps/templates/config.py

```diff
@@ -14,18 +14,24 @@ class ConfigTemplate(CMakeDepsFileTemplate):
 
     @property
     def filename(self):
-        if self.file_name == self.file_name.lower():
-            return "{}-config.cmake".format(self.file_name)
+        if self.find_module_mode:
```

Comment:
I thought that better a new class, ``FindTemplate`` instead of ``ConfigTemplate``, but then I saw the large amount of repeated code, and agree this is better implementation.